packages/composables/vitest.config.ts

```diff
@@ -7,14 +7,20 @@ export default defineConfig({
     coverage: {
       enabled: true,
       include: ["src"],
-      // "100": true, // TODO: our goal ;)
       thresholds: {
-        statements: 97,
-        branches: 90,
-        functions: 88.4,
-        lines: 97,
+        statements: 100,
+        branches: 100,
+        functions: 100,
+        lines: 100,
```

Comment:
we can replace it with that
```suggestion
        "100": true,
```